service/history/workflow/mutable_state_impl.go

```diff
@@ -36,6 +36,10 @@ import (
 
 	"github.com/nexus-rpc/sdk-go/nexus"
 	"github.com/pborman/uuid"
+	"google.golang.org/protobuf/proto"
```

Comment:
Same, fix your IDE configuration please, it adds noise in diffs.

Reply:
This is just a draft. Extra debugging logs. Will cleanup